src/features/virtualDocumentTracker.ts

```diff
@@ -63,7 +63,7 @@ function shouldIgnoreDocument(document: TextDocument, server: OmniSharpServer):
 }
 
 async function openVirtualDocument(document: TextDocument, server: OmniSharpServer, eventStream: EventStream) {
-    let req = { FileName: document.uri.path, changeType: FileChangeType.Create };
+    let req = { FileName: document.uri.fsPath, changeType: FileChangeType.Create };
```

Comment:
You need to fallback to `.path`. Aka,

```TypeScript
let path = document.uri.fsPath;

if (!path) {
    path = document.uri.path;
}
```

Same below.

Reply:
Done. Thanks!